### PR TITLE
Enhance comment doc to show new history feature

### PR DESCRIPTION
**Pull request by Yashsharma1911**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@Yashsharma1911 merge conflict.

**Review by leecalcote — COMMENTED** [review]
@nwanduka will you give consideration as to how this page and the other on commenting compliment one another? Are they entirely duplicative? Should we combine? Should we leave separate, but change their objectives slightly in order to better complement?

**Review by leecalcote — COMMENTED** [review]
@nwanduka will you give consideration as to how this page and the other on commenting compliment one another? Are they entirely duplicative? Should we combine? Should we leave separate, but change their objectives slightly in order to better complement?

**Comment by Yashsharma1911** [comment]
@nwanduka I added new screenshots, with good resolution and also fixed the merge conflict

**Comment by Yashsharma1911** [comment]
Screenshot 2023-12-07 at 2 03 29 PM

**Comment by Yashsharma1911** [comment]
Screenshot 2023-12-07 at 3 31 36 PM

**Review by nwanduka — COMMENTED** [review]
Hi @Yashsharma1911, the changes you made look good. You've done a great job combining the content of both pages. Just a few suggestions:
1. Could the width of the screenshots you added be reduced to be the same width as the other images on the page? For uniformity.
2. Under the "Resolve a Comment" subheading, you could add that users can reopen a resolved comment by commenting on it. Alternatively, as it appears the text under that heading is already getting longer than the others, we could consider adding another subheading (a number 7) that specifically highlights this feature. What do you think?

Besides these, I think the page is good to go. Unless @leecalcote has other feedback to offer.

File: content/en/meshmap/designer/comments/_index.md
[the remark this answers is not in the document]
Comment (leecalcote) [reply content/en/meshmap/designer/comments/_index.md:1]: @nwanduka, yes, you're right. This is a fully implemented feature, ready for complete documentation.

**Comment by Yashsharma1911** [comment]
Oops this got closed accidentally

**Comment by Yashsharma1911** [comment]
@iArchitSharma can you help ensuring we include these, lemme know I'll collaborate with you

**Comment by Yashsharma1911** [comment]
Reopening this PR

**Pull request closed without merging**